airflow/models.py

```diff
@@ -116,17 +114,24 @@ def __init__(
             self,
             dag_folder=None,
             executor=DEFAULT_EXECUTOR,
-            include_examples=configuration.getboolean('core', 'LOAD_EXAMPLES'),
```

Comment:
Should we replace this with a new accessor method ? (pas sur de pourquoi on le replace by None)